packages/user-interface/src/pages/MessagePage.tsx

```diff
@@ -1,59 +1,77 @@
-import { useGetTakenV2Query, TaakV2 } from "@nl-portal/nl-portal-api";
+import {
+  Bericht,
+  TaakV2,
+  useGetBerichtQuery,
+  useGetTakenV2Query,
+} from "@nl-portal/nl-portal-api";
 import { Paragraph } from "@gemeente-denhaag/typography";
 import { FormattedMessage } from "react-intl";
 import { useOutletContext, useParams } from "react-router-dom";
 import BackLink from "../components/BackLink";
 import PageGrid from "../components/PageGrid";
 import PageHeader from "../components/PageHeader";
 import TasksList from "../components/TasksList";
-import { MessageType } from "../components/Message";
-import { messages } from "./MessagesPage";
 import { RouterOutletContext } from "../contexts/RouterOutletContext";
+import styles from "./MessagePage.module.scss";
+import Skeleton from "../components/Skeleton.tsx";
 
 const MessagePage = () => {
   const { id } = useParams();
   const { paths } = useOutletContext<RouterOutletContext>();
-  const {
-    data: tasksData,
-    loading: taskLoading,
-    error: tasksError,
-  } = useGetTakenV2Query({
+  const { data: tasksData, loading: taskLoading } = useGetTakenV2Query({
     variables: { zaakId: id },
   });
-  const loading = taskLoading;
-  const error = tasksError;
-  const message: MessageType = messages.find((m) => m.id === Number(id))!;
-  const tasks = tasksData?.getTakenV2.content as TaakV2[] | undefined;
+  const {
+    data: messageData,
+    loading: messageLoading,
+    error: messageError,
+  } = useGetBerichtQuery({
+    variables: { id: id },
+  });
+  const tasks: TaakV2[] | undefined = tasksData?.getTakenV2.content as
```

Comment:
Don't think the first type declaration is needed if you use as.

Reply:
Fair, I only added it for easier readability.